include/devicetree.h

```diff
@@ -608,6 +608,24 @@
 #define DT_PROP_HAS_IDX(node_id, prop, idx) \
 	IS_ENABLED(DT_CAT6(node_id, _P_, prop, _IDX_, idx, _EXISTS))
 
+/**
+ * @brief Is name "name" available in a foo-names property?
```

Comment:
Can you give an example devicetree fragment and usage, like we do for other macros? This description is a bit abstract and I think an example would make it easier to use.

Reply:
Example added